docs/modules/security/pages/native-client-security.adoc

```diff
@@ -300,15 +300,50 @@ all:
 ----
 ====
 
+[[management-permission]]
+
+**Management Permission:**
+
+This permission defines which
+client principals/endpoints are allowed to perform management tasks
+(i.e. client supposed to be used by Hazelcast Management Center).
+

```

Comment:
This is the client that is started by Management Center when it connects to the clusters, right?

Reply:
Yes, that one :)